tools/generator/Tests/Unit-Tests/CodeGeneratorTests.cs

```diff
@@ -1,40 +1,27 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
 using generatortests.Unit_Tests;
 using MonoDroid.Generation;
 using NUnit.Framework;
-using System.Collections.Generic;
-using System.IO;
-using System.Linq;
-using System.Reflection;
-using System.Text;
+using Xamarin.Android.Binder;
 
 namespace generatortests
 {
-	abstract class CodeGeneratorTests
+	[TestFixture]
+	class JavaInteropCodeGeneratorTests : CodeGeneratorTests
```

Comment:
It seems odd that you're "moving" `JavaInteropCodeGeneratorTests` into `CodeGeneratorTests.cs` when it was already in `JavaInteropCodeGeneratorTests.cs`, and the previous "pattern" from e.g. 5ad6ae028c5d0a5dafc8c2494c94c3dadb01e5d7 was to place types into separate files...

Why the "file unification" here?